docs/decisions/0008-zbar-support-in-1.3.x.rst

```diff
@@ -0,0 +1,89 @@
+QR Scanning the native ZBar Dependency in 1.3.x
+###############################################
+
+:Date: 2023-05-22
+:Status: Completed.
+:Author: AustEcon
+
+Context
+-------
+Brief Summary
+^^^^^^^^^^^^^
+v1.3.15 has an issue with QR Scanning see: https://github.com/spesmilo/electrum/issues/6018.
```

Comment:
This is not a brief summary. It is confusing and complicated. Please state the problem and the solution in clear concise words.

Reply:
i.e. This is direct and clear example "The 1.3.x branches use zbar for the camera to scan QR codes. Zbar has bugs related to camera window scaling for some Windows users. While we could upgrade this support, we have decided not to."